File: Module-CP/Assignments/assignment14/Pass_by_value/q1.c
#include<stdio.h>
typedef struct Student{
    int rollNo;
    char name[20];
    int marks;
}Student;
Student storeStudent();
void display(Student);
int main(){
    Student s1,s2;
    s1=storeStudent();
    s2=storeStudent();
    printf("Details for S1:");
    display(s1);
    printf("Details for S2:");
    display(s2);
}
Student storeStudent(){
    Student s;
    printf("Enter Roll No: ");
    scanf("%d",&s.rollNo);
    printf("Enter Name: ");
    scanf("%s",s.name);
    printf("Enter Marks: ");
    scanf("%d",&s.marks);
    return s;
}
void display(Student s){
      printf("\nRoll No: %d",s.rollNo);
    printf("\nName: %s",s.name);
    printf("\nMarks: %d\n",s.marks);
}

File: Module-Core Java/Assignments/C to Java/Assignment 14/By_using_Functions/q1.java
import  java.util.*;
class Stdents{
    int rollNo;
    String name;
    int marks;

    Scanner sc = new Scanner(System.in);

    void setName(String name){
        this.name=name;
    }
    void setRollNo(int rollNo){
        this.rollNo=rollNo;
    }
    void setMarks(int marks){
        this.marks=marks;
    }
    

    String getName(){
        return this.name;
    }
    int getrollNo(){
        return this.rollNo;
    }
    int getMarks(){
        return this.marks;
    }
    void display(){
        System.out.println("Student Name:"+this.name);
        System.out.println("Student RollNum:"+this.rollNo);
        System.out.println("Student Marks:"+this.marks);
    }
}
public class q1{
    public static void main(String[] args) {
        Stdents s1 = new Stdents();
        Stdents s2 = new Stdents();

        s1.setName("Pratik");
        s1.setRollNo(27);
        s1.setMarks(97);
        s2.setName("Chetan");
        s2.setRollNo(20);
        s2.setMarks(90);

        System.out.println("Details Of Student-1");
        s1.display();

        System.out.println("Details Of Student-2");
        s2.display();
    }
}